library/core/src/unicode/mod.rs

```diff
@@ -31,3 +31,55 @@ mod unicode_data;
 /// [Unicode 11.0 or later, Section 3.1 Versions of the Unicode Standard](https://www.unicode.org/versions/Unicode11.0.0/ch03.pdf#page=4).
 #[stable(feature = "unicode_version", since = "1.45.0")]
 pub const UNICODE_VERSION: (u8, u8, u8) = unicode_data::UNICODE_VERSION;
+
+#[cfg(kani)]
+mod verify {
+    use super::conversions::{to_upper, to_lower};
+    use wrappers::*;
+    use crate::kani;
+    use safety::*;
+
+    /// Wrapper functions used to verify auto-generated functions from this module.
+    ///
+    /// The files in this module are auto-generated by a script, so they are harder to annotate.
+    /// Instead, for each function that we want to verify, we create a wrapper function with
+    /// contracts.
```

Comment:
Wouldn't it be possible to add the annotations to the script so that the functions plus their contracts are auto-generated? This would avoid the problem that we could never use these contracts as abstractions for no code will ever use the wrapper functions.

Reply:
I agree that it would be ideal, but the script is not currently hosted in our repository and it is not run as part of the verification flow. This is a stopgap for now, and once we are ready to push this upstream, the annotation should go on the tool that generates this contract.

Reply:
Thank you for clarifying! Could you perhaps include some information on what that script is or where it is located in the comment?

Reply:
Sure... that comment is on the top of the generated `rs` files, but I can add them to this doc too.